rules/0131.go

```diff
@@ -0,0 +1,202 @@
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/googleapis/api-linter/rules/descriptor"
+	p "google.golang.org/protobuf/reflect/protoreflect"
+)
+
+func init() {
+	registerRules(
+		checkGetRequestMessageName(),
+		checkGetRequestMessageNameField(),
+		checkGetRequestMessageUnknownFields(),
+		checkGetResponseMessageName(),
+	)
+}
+
+// Get messages should have a properly named Request message.
+func checkGetRequestMessageName() lint.Rule {
+	return &descriptor.CallbackRule{
+		RuleInfo: lint.RuleInfo{
+			Name:         lint.NewRuleName("core", "0131", "request_message", "name"),
+			Description:  "check that Get RPCs have appropriate request messages",
+			RequestTypes: []lint.RequestType{lint.ProtoRequest},
+			URI:          "https://aip.dev/131#guidance",
+		},
+		Callback: descriptor.Callbacks{
+			MethodCallback: func(m p.MethodDescriptor, s lint.DescriptorSource) (problems []lint.Problem, err error) {
+				// We only care about Get methods for the purpose of this rule;
+				// ignore everything else.
+				if !isGetMethod(m) {
+					return
+				}
+
+				// Rule check: Establish that for methods such as `GetFoo`, the request
+				// message is named `GetFooRequest`.
+				methodName := string(m.Name())
+				requestMessageName := string(m.Input().Name())
+				if requestMessageName != methodName+"Request" {
+					problems = append(problems, lint.Problem{
+						Message: fmt.Sprintf(
+							"Get RPCs should have a request message named after the RPC, such as %q.",
+							methodName+"Request",
+						),
+						Suggestion: methodName + "Request",
+						Descriptor: m,
+					})
+				}
+
+				return problems, nil
+			},
+		},
+	}
+}
+
+// The Get standard method should only have expected fields.
+func checkGetRequestMessageNameField() lint.Rule {
+	return &descriptor.CallbackRule{
+		RuleInfo: lint.RuleInfo{
+			Name:         lint.NewRuleName("core", "0131", "request_message", "name_field"),
+			Description:  "check that a name field is present",
+			RequestTypes: []lint.RequestType{lint.ProtoRequest},
+			URI:          "https://aip.dev/131#request-message",
+		},
+		Callback: descriptor.Callbacks{
+			MethodCallback: func(m p.MethodDescriptor, s lint.DescriptorSource) (problems []lint.Problem, err error) {
+				// We only care about Get methods for the purpose of this rule;
+				// ignore everything else.
+				if !isGetMethod(m) {
+					return
+				}
+
+				// Rule check: Establish that a name field is present.
+				nameField := m.Input().Fields().ByName("name")
+				if nameField == nil {
+					problems = append(problems, lint.Problem{
+						Message:    fmt.Sprintf("method %q has no `name` field", m.Name()),
+						Descriptor: m.Input(),
+					})
+					return problems, nil
+				}
+
+				// Rule check: Establish that the name field is a string.
+				if nameField.Kind() != p.StringKind {
+					problems = append(problems, lint.Problem{
+						Message:    "`name` field on Get RPCs should be a string",
+						Descriptor: nameField,
```

Comment:
Same question here -- where should the lint warning show up?

Reply:
On the field is intentional and correct.